res/controllers/Hercules-DJ-Console-RMX-hid-scripts.js

```diff
@@ -217,6 +313,35 @@ RMX.init = function() {
     function(g, e, v) { c.send(g, "headphone_cue", v);
   });
 
+
+  RMX.shutdown = function() {
+    var c = RMX; 
+    for (id in c.leds) {
+      c.send(c.leds[id].group, c.leds[id].name, 0);
+    }
+  }
+
+  // Enable soft-takeover for all direct hardware controls
+  // engine.softTakeover("[Channel1]","rate",true);
+  engine.softTakeover("[Channel1]","volume",true);
+  engine.softTakeover("[Channel1]","pregain",true);
+  engine.softTakeover("[Channel1]","filterHigh",true);
+  engine.softTakeover("[Channel1]","filterMid",true);
+  engine.softTakeover("[Channel1]","filterLow",true);
+  // engine.softTakeover("[Channel2]","rate",true);
+  engine.softTakeover("[Channel2]","volume",true);
+  engine.softTakeover("[Channel2]","pregain",true);
+  engine.softTakeover("[Channel2]","filterHigh",true);
+  engine.softTakeover("[Channel2]","filterMid",true);
+  engine.softTakeover("[Channel2]","filterLow",true);
+  // engine.softTakeover("[Master]","crossfader",true);
+  // engine.softTakeover("[Master]","headVolume",true);
+  // engine.softTakeover("[Master]","headMix",true);
+  // engine.softTakeover("[Master]","volume",true);
+  // engine.softTakeover("[Master]","balance",true);
```

Comment:
Could you please remove this comment cruft? Note that `[Master]` group is also deprecated (although I cannot find the documentation about this anymore...)